content/docs/command-reference/import-url.md

```diff
@@ -336,3 +351,46 @@ $ dvc repro
 Running stage 'prepare' with command:
 	python src/prepare.py data/data.xml
 ```
+
+## Example: Transfer to remote storage
+
+When you have a large dataset in an external location, you may want to import it
+to you project without downloading it to the local file system (for using it
+later/elsewhere). The `--to-remote` option lets you skip the download, while
+storing the imported data [remotely](/doc/command-reference/remote). Let's
+initialize a DVC project, and setup a remote:
+
+```dvc
+$ mkdir example # workspace
+$ mkdir /tmp/dvc-storage
+$ cd example
+$ git init
+$ dvc init
+$ dvc remote add myremote /tmp/dvc-storage
+```
+
+Now let's create an import `.dvc` file without downloading the target data,
+transferring it directly to remote storage instead:
+
+```
+$ dvc import-url https://data.dvc.org/get-started/data.xml data.xml \
+                 --to-remote -r local_remote
+...
+```
+
+When you run the `import-url` with `--to-remote`, you pass as usual the remote
+location and the output filename, afterward if you haven't set a default
+[remote](/doc/command-reference/remote) yet, you can simply pass the name of the
+remote with `-r`/`--remote` flag and it will start the transfer and leave a DVC
+file as an only side effect on your workspace (everything else happens in the
+remote storage unit)
```

Comment:
````suggestion
```
````

After the previous changes we did I think all of this is redundant, and the default remote stuff can be explained in the Options section.

Reply:
BUT I think we could extend the code block (or add a short sentence and another code block) to show what happened in the workspace (only .dvc file created, nothing cached or linked) and in the remote (imported data is stored there).